tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
Why revert it? The default fix for this rule should be the same as the default spacing on jsx-space-before-closing - which is to include the space.